tests/src/test/scala/scalaz/IListTest.scala

```diff
@@ -138,6 +134,10 @@ object IListTest extends SpecLite {
     ns.collectFirst(pf).toOption must_=== ns.toList.collectFirst(pf)
   }
 
+  "concat" ! forAll { (ns: IList[Int], ms: IList[Int]) =>
+    (ns concat ms).toList must_=== ns.toList ++ ms.toList
+  }
+
   "concat" ! forAll { (ns: IList[Int], ms: IList[Int]) =>
     (ns ++ ms).toList must_=== ns.toList ++ ms.toList
   }
```

Comment:
This one is also a duplicate of the test from line 96. (But I see that it was not you who introduced this one.)

Reply:
Yes, firstly I wanted to get rid of it, but then I realized it might test that `concat` behaves same way as `++`. I do not see any problem with deleting it.

Reply:
I mean, they are exactly the same, both testing `++`.

Having one test for `concat` and a similar test for `++` is fine.